bin/backstop

```diff
@@ -20,19 +20,28 @@ ROOT_DIR=$(dirname "$BIN_DIR")
 VERSION=6.1.1
 IMAGE=googlesitekit/vrt:$VERSION
 
-# Build the backstopjs image if it doesn't exist yet.
-docker image inspect $IMAGE -f "{{.Id}}" >/dev/null 2>&1 ||
-	docker build -t $IMAGE --build-arg BACKSTOPJS_VERSION=$VERSION "$ROOT_DIR/docker/backstopjs"
+# (re)Build the backstopjs image as needed.
+docker build \
+	--pull \
+	--tag $IMAGE \
+	--build-arg BACKSTOPJS_VERSION=$VERSION \
+	"$ROOT_DIR/docker/backstopjs"
 
 # Run the backstopjs command.
-docker run --rm -t --mount type=bind,source="$ROOT_DIR",target="/src" $IMAGE "$1" --config=/src/tests/backstop/config.js
+docker run --rm --tty \
+	--shm-size=512m \
+	--mount "type=bind,source=$ROOT_DIR,target=/src" \
+	$IMAGE \
+	"$@" \
```

Comment:
This also fixes an issue where only the command was being passed through. This passes everything so that it's possible to use other flags such as `--filter Modules/Analytics4/Widgets/DashboardOverallPageMetricsWidgetGA4/LoadedEntityURL` to only run a specific scenario.

Reply:
This is great!